app/src/styles/base.css

```
@@ -579,3 +582,45 @@ h1, h2, h3, h4 { font-family: var(--font-heading); margin: 0; }
   border-top: 0 !important;
 }
+
+
+/* The install demo (components/InstallDemo.tsx): a small phone that plays the
+ * tap sequence on a loop, because an arrow can only point at a toolbar — it
+ * cannot say which of its buttons, or what the next screen looks like.
+ *
+ * Three scenes share one 9s timeline and are offset by negative delays
+ * (0s / -3s / -6s) rather than being driven from JS: no timers to start, stop
+ * or leak, and every copy of the demo on screen stays in step. */
+@keyframes installScene {
```

Stylelint (17.14.0)

[error] Expected keyframe name "installScene" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)